<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Option</class>
 <widget class="QWidget" name="Option">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>481</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>571</width>
    <height>481</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>571</width>
    <height>481</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Настройки</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>491</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="font_select_label">
        <property name="maximumSize">
         <size>
          <width>280</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Шрифт:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="font_select_comboBox">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <item>
         <property name="text">
          <string>Consolas</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Courier</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Courier New</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Lucida Console</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="font_color_label">
        <property name="maximumSize">
         <size>
          <width>280</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Цвет текста:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="font_color_button">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Выбрать цвет</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="font_correct_color_label">
        <property name="maximumSize">
         <size>
          <width>280</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Цвет правильного текста:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="font_correct_color_button">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Выбрать цвет</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="font_wrong_color_label">
        <property name="maximumSize">
         <size>
          <width>280</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Цвет неправильного текста:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="font_wrong_color_button">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Выбрать цвет</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="input_field_color_label">
        <property name="maximumSize">
         <size>
          <width>280</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Цвет поля вывода:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="input_field_color_button">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Выбрать цвет</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="sound_label">
        <property name="maximumSize">
         <size>
          <width>167</width>
          <height>260</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Звук:</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QCheckBox" name="sound_wrong_input_checkBox">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>неправильный
ввод</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="sound_wrong_input_comboBox">
            <property name="minimumSize">
             <size>
              <width>130</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>16777215</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>wrong_sound1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>wrong_sound2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>wrong_sound3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>wrong_sound4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>wrong_sound5</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QCheckBox" name="sound_menu_checkBox">
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>кнопки меню</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="sound_menu_comboBox">
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>20</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>click_sound1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>click_sound2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>click_sound3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>click_sound4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>click_sound5</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>430</y>
     <width>158</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QPushButton" name="accept_button">
      <property name="text">
       <string>Применить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_button">
      <property name="text">
       <string>Отмена</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
